queries.sql

```diff
@@ -181,6 +181,10 @@ WHERE vt.name = 'Maisy Smith'
 GROUP BY s.name
 ORDER BY visit_count DESC
 LIMIT 1;
+
+SELECT COUNT(*) FROM visits where animal_id = 4;
+SELECT * FROM visits where vet_id = 2;
+SELECT * FROM owners where email = 'owner_18327@mail.com';
```

Comment:
- [ ] 👍 Good job on providing a screenshot of your query! However, it would be even more informative if you could include the full query, including the `EXPLAIN ANALYZE` part that you used in the command line. This will give a complete view of your query's execution. Please consider updating your `queries.sql` file and include these details. Keep up the great work! 📊👏
![image](https://github.com/Kaiserabbas/vet_clinic/assets/55840999/63131fbf-eabe-44d1-ae09-a336a029c2a0)

Reply:
https://github.com/Kaiserabbas/vet_clinic/blob/29ad7e8c30ea1637495195c1818c381860d7dfaf/queries.sql#L185 
https://github.com/Kaiserabbas/vet_clinic/blob/29ad7e8c30ea1637495195c1818c381860d7dfaf/queries.sql#L186 
https://github.com/Kaiserabbas/vet_clinic/blob/29ad7e8c30ea1637495195c1818c381860d7dfaf/queries.sql#L187 
https://github.com/Kaiserabbas/vet_clinic/blob/29ad7e8c30ea1637495195c1818c381860d7dfaf/queries.sql#L189